README.md

```diff
@@ -72,7 +72,7 @@ Example Playbook
         - role: rhel7-stig
           when:
             - ansible_os_family == 'RedHat'
-            - ansible_distribution_major_version | version_compare('7', '=')
+            - ansible_distribution_major_version is version('7', '==')
```

Comment:
No. The alias is not deprecated, and version doesn't work on older ansible versions.

Reply:
Understood - I'll revert to version_compare and resubmit the PR.